types/backbone/index.d.ts

```diff
@@ -293,34 +293,32 @@ declare namespace Backbone {
          * persisted to the server. Unset attributes will be set to undefined.
          * You can also pass an attributes object to diff against the model,
          * determining if there *would be* a change. */
-        changedAttributes(attributes?: any): any;
-        clear(options?: Silenceable): any;
+        changedAttributes(attributes?: Partial<T>): Partial<T> | false;
+        clear(options?: Silenceable): this;
         clone(): Model;
-        destroy(options?: ModelDestroyOptions): any;
-        escape(attribute: string): string;
-        has(attribute: string): boolean;
-        hasChanged(attribute?: string): boolean;
+        destroy(options?: ModelDestroyOptions): JQueryXHR | false;
+        escape(attribute: keyof T & string): string;
```

Comment:
Why not just `keyof T`?

Reply:
Because when `T = any`, `keyof T = any` too, but attr name should be always a string